realm/src/main/java/io/realm/Realm.java

```diff
@@ -955,6 +955,35 @@ boolean contains(Class<?> clazz) {
         return new RealmResults<E>(this, tableView, clazz);
     }
 
+    /**
+     * Get all objects of a specific Class sorted by specific field names.
+     *
+     * @param clazz the Class to get objects of.
+     * @param fieldNames an array of field names to sort by.
+     * @param sortAscending sort ascending if SORT_ORDER_ASCENDING, sort descending if SORT_ORDER_DESCENDING.
+     * @return  A sorted RealmResults containing the objects.
+     * @throws java.lang.IllegalArgumentException if a field name does not exist.
+     */
+    public <E extends RealmObject> RealmResults<E> allObjects(Class<E> clazz, String fieldNames[], boolean sortAscending) {
```

Comment:
Gah, forgot about that.
That means it is impossible to create a really nice interface for both multi column and single column sort.
I think I would prefer 3 methods then (The first 2 probably covers 90+% of usecases):
```
public void sort(String fieldName, boolean sortAscending); // Current 
public void sort(String f1, boolean s1, String f2, boolean s2); // Most common multisort case. Not sure about good names
public void sort(String[] fieldNames, boolean[] sortAscending); // General case.

What do you think?